### PR TITLE
[87] Add django management commands for specializations and units

### PR DESCRIPTION
**Pull request by CatarinaSilva**
From readme:

> Create a specialization:
> ```
> docker-compose run --rm django ./manage.py create-spec \
>   -c S01 \
>   -n 'Bootcamp'
> ```
> 
> Create a unit:
> 
> ```
> docker-compose run --rm django ./manage.py create-unit \
>   -s S01 \
>   -c SLU01 \
>   -n 'SLU01 - Pandas' \
>   -due '2020-04-10' \
>   --open
> ```

If everything goes well you should see:

Screenshot 2021-08-22 at 19 26 23

If the unit or spec already exists:

Screenshot 2021-08-22 at 19 24 12

### CODE REVIEW

File: portal/academy/models.py [hunk 1]
```diff
@@ -11,11 +9,17 @@ class Specialization(models.Model):
     code = models.CharField(max_length=255, primary_key=True)
     name = models.CharField(max_length=255)
     description = models.TextField(blank=True)
-    created = models.DateTimeField(auto_now_add=True)
+    created = models.DateTimeField(editable=False)
```
Comment (hcastilho) [inline portal/academy/models.py:12]: Is this some change that ended up not being used?

File: portal/academy/models.py [hunk 2]
```diff
@@ -11,11 +9,17 @@ class Specialization(models.Model):
...
 
     def __str__(self):
         return self.code
 
+    def save(self, *args, **kwargs):
+        ''' On save for the first time, set created timestamp'''
+        if not self.created:
+            self.created = timezone.now()
+        return super(Specialization, self).save(*args, **kwargs)
```
Comment (hcastilho) [inline portal/academy/models.py:21]: No need to pass the class and the class instance to super.

File: portal/academy/models.py (hunk 1 above)
Comment (CatarinaSilva) [reply portal/academy/models.py:12]: the `auto_now_add` was not working properly. I searched a bit and found some recommendations to use `editable=False` + the custom save function)

File: portal/academy/models.py (hunk 2 above)
Comment (CatarinaSilva) [reply portal/academy/models.py]: changed it accordingly :)

File: portal/academy/models.py (hunk 1 above)
Comment (CatarinaSilva) [reply portal/academy/models.py]: Reversed these changes  as discussed via slack 😉

**Review by hcastilho — APPROVED** [review]

**Pull request merged**